### PR TITLE
[Improvement](regresstion-test)Enhance regression strategy and  Allows load and query to run simultaneously in different directories

### PR DESCRIPTION
**Pull request by zhangguoqiang666**
# Proposed changes

## Problem summary

Describe your changes.
  Enhance regression strategy and  Allows load and query to run simultaneously in different directories
  add code to RegressionTest.groovy
## Checklist(Required)

* [no ] Does it affect the original behavior
* [no] Has unit tests been added
* [no] Has document been added or modified
* [no] Does it need to update dependencies
* [yes] Is this PR support rollback (If NO, please explain WHY)

## Further comments
  This pr may affect the performance of regression testing, especially for non-standard regression use cases

### CODE REVIEW

**Comment by zhangguoqiang666** [comment]
run buildall

File: regression-test/framework/src/main/groovy/org/apache/doris/regression/RegressionTest.groovy [hunk 1]
```diff
@@ -180,13 +180,43 @@ class RegressionTest {
...
+            List<String> load_sources = new ArrayList<>()
+            List<String> other_sources = new ArrayList<>()
+            new File(config.suitePath).eachDir { dir ->
+                {
+                    def load_flag = 0
+                    dir.eachFileRecurse { f_01 ->
+                        if (f_01.name.contains("load")) {
+                            load_flag = 1
+                            load_sources.add(f_01.name)
+                        }
```
Comment (gavinchou) [inline regression-test/framework/src/main/groovy/org/apache/doris/regression/RegressionTest.groovy:192]: Can we just use the following pattern to classify the script files?
```
if (contains(load) load.add(f)
else other.add(f)
```

**Comment by zhangguoqiang666** [comment]
run buildall

**Comment by zhangguoqiang666** [comment]
run buildall

**Comment by zhangguoqiang666** [comment]
run compile

**Comment by zhangguoqiang666** [comment]
run p0

**Comment by zhangguoqiang666** [comment]
run compile

**Comment by zhangguoqiang666** [comment]
run compile

**Comment by zhangguoqiang666** [comment]
run compile

**Comment by zhangguoqiang666** [comment]
run compile

**Comment by zhangguoqiang666** [comment]
run buildall

**Comment by zhangguoqiang666** [comment]
run p0

**Comment by zhangguoqiang666** [comment]
run p0

**Pull request closed without merging**